package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/stealjs/steal-less",
   "dependencies": {
-    "less": "2.4.0 - 2.5.3",
+    "less": "2.6.0",
```

Comment:
I think it’s a breaking change to set this just to `2.6.0`; maybe the range could be widened?

I wonder why 2.6 wasn’t included when this range was set: https://github.com/stealjs/steal-less/pull/61/files

Reply:
I think at the time that was originally written 2.5.3 was the latest and that pull you're referencing is a simple revert to that.